modules/javafx.controls/src/main/java/javafx/scene/control/Labeled.java

```diff
@@ -818,6 +828,37 @@ public final BooleanProperty mnemonicParsingProperty() {
         return mnemonicParsing;
     }
 
+    /**
+     * Indicates whether the text has been truncated
+     * because it cannot fit into the available width.
+     * <p>
+     * When truncated, the {@link #ellipsisStringProperty() ellipsisString}
+     * gets inserted in the place dictated by the
+     * {@link #textOverrun} property.
```

Comment:
I just noticed that this link doesn't work. Looks like you'll need to change it to point to the `textOverrunProperty()` method.